src/main/markdown/eco/eco-system.md

```diff
@@ -0,0 +1,263 @@
+# GWT Eco System
+
+Today, the GWT eco system is not only that, what comes with the framework. Usually, projects use a lot of third party
+dependencies to improve there development experience, add features or prefer a different approach to do things. 
+
+But, before we start a little history of GWT, the current state and a foresight:
+
+## History
+
+First and foremost, GWT is a Java to JavaScript transpiler. The basic idea is to use a type-safe and object oriented
+language to create applications which can ran natively in the browser. During development, developers benefit from the 
+existing Java tool chain, powerful IDEs and awesome refactoring abilities. In production the transpiler creates highly
+optimized and obfuscated JavaScript.
+
+In 2006, as GWT emerges, things were different. JavaScript depends on the browser and Java were in an
+early state of evolution. That's one of the reasons why we have permutations and generators in GWT. Also there no
```

Comment:
I don't quite understand "...and Java [was] in an early state of evolution" - is this referring to generators rather than apt? If so, disregard.

Reply:
Yeah, that is my intention.